samples/smartsyncexplorer/SmartSyncExplorer.html

```diff
@@ -360,8 +347,7 @@ <h1 class="title">Contacts</h1>
 
     setupCaches: function() {
         // Cache for offline support
-        app.cache = new Force.StoreCache("contacts", [ {path:"FirstName", type:"full_text"}, {path:"LastName", type:"full_text"}]);
-        app.cache.init();
+        app.cache = new Force.StoreCache("contacts" /* see userstore.json */);
```

Comment:
We don't call app.cache.init() anymore (which would register the soup since the soups should have been setup already from the config file).